components/Buttons/Button.js

```diff
@@ -1,8 +1,8 @@
 import Link from "next/link";
 
-const Button = ({ caption, link }) => {
+const Button = ({ caption }) => {
   return (
-    <Link href={link} className="items-center">
+    <Link href="#categories" className="items-center">
```

Comment:
How come this is being hard coded? Do you not reuse this button at all?

Reply:
Hi @ARTurnerGit. It makes perfect sense,  I don't know what I was thinking! Fixed.